awx/main/models/schedules.py

```diff
@@ -88,6 +88,15 @@ class Meta:
     def get_zoneinfo(self):
         return sorted(get_zonefile_instance().zones)
 
+    @classmethod
+    def get_zoneinfo_with_links(self):
```

Comment:
Instead of returning a dict like `{"zones": {...}, "links": {...}}`, it might be more clear to return a tuple of `(zones, links)`. That way the specific dict structure is more view-specific. This is something devs are can be pretty opinionated about, so I offer this as a very weak suggestion.